### DISCUSSION TITLE
关于直播间的进度同步功能

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#1744] [功能建议 / Ideas]
前几个月直播间播放器改版前出了个追进度的功能，改版后被砍了，但是手机端还留着

原理大概就是倍速播放直至达到同步到最新进度

今天发现斗鱼也有第三方脚本实现了这个功能，不过貌似不是单纯通过倍速实现，而是提供一个按钮，点了直接跳转到已缓存的最新帧

**Comment by the1812** [comment]
之前我记得有个人提了 不过需要挺长时间写和测试（

**Reply by u2shana** [reply]
https://github.com/qianjiachun/douyuEx/blob/master/douyuex.js

你可以参考下这脚本

**Reply by Geeyun-JY3** [reply]
在这：#1342
